### PR TITLE
feat: synchronize audit log login attempts from b2c

### PR DESCRIPTION
**Pull request by michaelmajgaard**

### CODE REVIEW

File: source/marketparticipant/Energinet.DataHub.MarketParticipant.Infrastructure/Persistence/Repositories/CutoffRepository.cs [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+        {
+            return await UpdateCutoffAsync(type, Instant.FromUnixTimeTicks(0)).ConfigureAwait(false);
+        }
+
+        return cutoff.Timestamp.ToInstant();
+    }
+
+    public async Task<Instant> UpdateCutoffAsync(CutoffType type, Instant timestamp)
+    {
+        var cutoff = await _context.Cutoffs.SingleOrDefaultAsync(x => x.Type == (int)type).ConfigureAwait(false);
```
Comment (vbelinschi) [inline source/marketparticipant/Energinet.DataHub.MarketParticipant.Infrastructure/Persistence/Repositories/CutoffRepository.cs:48]: why search for it again, if it returned `null` just before?

File: source/marketparticipant/Energinet.DataHub.MarketParticipant.Application/Commands/AuditLogLoginAttemptLogEntriesCommand.cs [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using MediatR;
+
+namespace Energinet.DataHub.MarketParticipant.Application.Commands;
+
+public sealed record AuditLogLoginAttemptLogEntriesCommand : IRequest;
```
Comment (defectiveAi) [inline source/marketparticipant/Energinet.DataHub.MarketParticipant.Application/Commands/AuditLogLoginAttemptLogEntriesCommand.cs:19]: AuditLoginAttemptsCommand

File: source/marketparticipant/Energinet.DataHub.MarketParticipant.Infrastructure/Persistence/Repositories/CutoffRepository.cs (hunk 1 above)
Comment (michaelmajgaard) [reply source/marketparticipant/Energinet.DataHub.MarketParticipant.Infrastructure/Persistence/Repositories/CutoffRepository.cs:48]: Fixed

File: source/marketparticipant/Energinet.DataHub.MarketParticipant.Application/Commands/AuditLogLoginAttemptLogEntriesCommand.cs (hunk 2 above)
Comment (michaelmajgaard) [reply source/marketparticipant/Energinet.DataHub.MarketParticipant.Application/Commands/AuditLogLoginAttemptLogEntriesCommand.cs]: Fixed

**Review by defectiveAi — APPROVED** [review]

**Pull request merged**